src/SFA.DAS.AdminService.Web/ViewModels/CertificateDelete/CertificateConfirmDeleteViewModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Newtonsoft.Json;
+using SFA.DAS.AssessorService.Domain.Consts;
+using SFA.DAS.AssessorService.Domain.Entities;
+using SFA.DAS.AssessorService.Domain.JsonData;
+
+namespace SFA.DAS.AdminService.Web.ViewModels.CertificateDelete
+{
+    public class CertificateConfirmDeleteViewModel : CertificateBaseViewModel, ICertificateViewModel
+    {
+        public long Uln { get; set; }
+        public string CertificateReference { get; set; }
+        public int StandardCode { get; set; }
+        public string IncidentNumber { get; set; }
+        public string UserName { get; set; }
+        public void FromCertificate(Certificate cert)
```

Comment:
Method FromCertificate() used in CertificateAuditDetailsViewModel  and CertificateConfirmDeleteViewModel , can you  either move to base class or helper class.